common/planners/src/plan_replace.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_datavalues::DataSchemaRef;
+use common_meta_types::MetaId;
+
+use crate::PlanNode;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone)]
+pub struct ReplacePlan {
+    pub db_name: String,
+    pub tbl_name: String,
+    pub tbl_id: MetaId,
```

Comment:
For structure fields, always using full-name e.g., `table_name`, `table_id` would be better for searching etc.

Abbreviations should only be considered when being used as local variable names.

Reply:
Agree

Reply:
thanks for guiding : )